pyproject.toml

```diff
@@ -1,6 +1,54 @@
+[project]
+name = "wagtail-flags"
+version = "5.3.0"
```

Comment:
Should we target a major release at 6.x since this drops support for older versions?

Reply:
Why not?